package.json

```diff
@@ -32,13 +32,14 @@
     "@babel/runtime": "^7.19.0",
     "@juliushaertl/vue-richtext": "^1.0.1",
     "@nextcloud/auth": "^2.0.0",
-    "@nextcloud/axios": "^2.0.0",
+    "@nextcloud/axios": "^1.11.0",
```

Comment:
Seems this needs a rebase to not downgrade the version ;)

Reply:
since the [dependency issue with the notify_push module](https://github.com/icewind1991/notify_push-client/pull/5) is now fixed, I've reverted the [commit with the workaround](https://github.com/nextcloud/deck/commit/5aa36aa5ecb84cb576a3c226c0368a73a1ccfa3a) and therefore not downgrading the `@nextcloud/axios` anymore ;)